kibana/values.yaml

```diff
@@ -31,6 +31,13 @@ resources:
     cpu: "1000m"
     memory: "1Gi"
 
+# If Pod Security Policy in use it may be required to specify security context as well as service account
+securityContext:
+  #runAsUser: "place the user id here"
+  #fsGroup: "place the group id here"
+ 
+serviceAccount: "default"
```

Comment:
With `{{- if .Values.serviceAccount }}` being set in the template this will always be enabled by default. Can you instead make the default that it is undefined? I would rather have users opt in for using a service account rather than accidentally giving Kibana more access than it needs.